src/cli/commands/build/build.ts

```diff
@@ -37,6 +39,15 @@ export async function build(options: SWACLIConfig) {
   if (options.auto && hasBuildOptionsDefined(options)) {
     logger.error(`You can't use the --auto option when you have defined appBuildCommand or apiBuildCommand in ${swaCliConfigFilename}`);
     logger.error(`or with the --app-build-command and --api-build-command options.`, true);
+    const cmdEndTime = new Date().getTime();
+
+    collectTelemetryEvent(TELEMETRY_EVENTS.Build, {
+      duration: (cmdEndTime - cmdStartTime).toString(),
+      flagsUsed: JSON.stringify(flagsUsed),
```

Comment:
Do we want to keep it first level? configFlagsUsed, userFlagsUser ?

Reply:
Yes this is what @Reshmi-Sriram suggested.